src/js/InterfaceManager.js

```diff
@@ -0,0 +1,232 @@
+import { NavLink } from "react-router-dom";
+/**
+ * Get an array of prepared tracks to insert into HTML.
+ * @param  {} array - Array with tracks formed by spotify.js
+ * @param  {} limit=-1 - Limit of tracks
+ */
+export function getSearchTracksHTML(array, limit = -1) {
+    let result = [];
+
+    for (let i = 0; i < array.length; i++) {
+        if (result.length >= limit && limit !== -1)
+            break;
+        result.push(<div className="song-item" key={i}>
```

Comment:
В качестве ключа лучше использовать что-то другое.
Например id записи если он есть
```suggestion
        result.push(<div className="song-item" key={array[i].id}>
```

Reply:
Готово